Content.Server/GameObjects/Components/GUI/InventoryComponent.cs

```diff
@@ -175,9 +175,9 @@ public bool CanEquip(Slots slot, ItemComponent item, out string reason)
         /// </summary>
         /// <param name="slot">The slot to drop the item from.</param>
         /// <returns>True if an item was dropped, false otherwise.</returns>
-        public bool Unequip(Slots slot)
+        public bool Unequip(Slots slot, bool force = false)
```

Comment:
New params could use doc comments.

Reply:
Will do a doc pass at some point, probably tonight since I'll be fixing some stuff (and maybe the refactor suggested above too) anyway.